runtime/kusama/Cargo.toml

```diff
@@ -19,6 +19,7 @@ smallvec = "1.6.1"
 authority-discovery-primitives = { package = "sp-authority-discovery", git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 babe-primitives = { package = "sp-consensus-babe", git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 beefy-primitives = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
+kusama-runtime-constants = { package = "kusama-runtime-constants", path = "./constants", default-features = false }
```

Comment:
You need to enable the std feature below in the file.

Reply:
Done